gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/shift/SweNetworkShifter.java

```diff
@@ -126,6 +126,7 @@ public void shiftNetwork(double stepValue, Network network) throws GlskLimitatio
                 LoadFlowResult result = LoadFlow.run(network, workingVariantCopyId, LocalComputationManager.getDefault(), LoadFlowParameters.load());
                 if (!result.isOk()) {
                     LOGGER.error("Loadflow computation diverged on network '{}'", network.getId());
```

Comment:
I think it's bette to add the direction in this message LOGGER.error because it's not added automatically like with Business logger